docker-compose.yml

```diff
@@ -1,5 +1,11 @@
 version: '3.8'
 services:
+  dev:
```

Comment:
I don't understand what is the purpose of the current _dev service_. It starts a node container and shows my current etc/host. 

If we want to create a development container I would suggest modifying the current _start service_ with

- add volume mount
- rename service in dev

```
  dev:
    build:
      context: .
    ports:
      - 3000:3000
    user: $UID:$GID
    volumes:
      - .:/app
```

Reply:
Chiarito nel commento.